library/Icinga/Web/Response.php

```diff
@@ -357,6 +368,19 @@ protected function prepare()
             }
         }
 
+        $useCsp = (bool) Config::app()->get('security', 'use_csp', false);
+
+        if ($useCsp) {
+            $this->setHeader('Strict-Transport-Security', 'max-age=31536000; includeSubdomains; preload', true);
```

Comment:
Why all the other headers?

Reply:
Could this be clarified in the docs? Possibly linking to it in the description when setting up CSP? Since this seems to be best practice.